crates/cxx-qt-lib/src/gui/qfont.rs

```diff
@@ -0,0 +1,222 @@
+// SPDX-FileCopyrightText: 2024 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Laurent Montel <laurent.montel@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+use cxx::{type_id, ExternType};
+use std::mem::MaybeUninit;
+
+#[cxx::bridge]
+mod ffi {
+
+    #[repr(i32)]
+    #[namespace = "rust::cxxqtlib1"]
+    #[derive(Debug)]
+    enum QFontStyle {
+        StyleNormal,
+        StyleItalic,
+        StyleOblique,
+    }
+
+    /// This enum describes the different levels of hinting that can be applied to glyphs
+    /// to improve legibility on displays where it might be warranted by the density of pixels.
+    #[repr(i32)]
+    #[namespace = "rust::cxxqtlib1"]
+    #[derive(Debug)]
+    enum QFontHintingPreference {
+        PreferDefaultHinting,
```

Comment:
can you add doc lines to these too (just copy Qt docs). As eventually we want to turn missing docs into errors in cxx-qt-lib